src/components/S/IAddress.vue

```diff
@@ -5,15 +5,15 @@ export default {
 </script>
```

Comment:
Any reason we have script doubled here and on others files?

Reply:
We need to export inheritAttrs and it's only possible in non-setup script, so in this case we need one regular script and other (typical) setup script.